pipelines/build/openjdk11_pipeline.groovy

```diff
@@ -55,27 +55,27 @@ def buildConfigurations = [
                 os                  : 'linux',
                 arch                : 's390x',
                 additionalNodeLabels: 'build-marist-rhel74-s390x-2',
-                test                : ['openjdktest', 'systemtest']
+                test                : ['openjdktest', 'systemtest', 'perftest']
```

Comment:
has this been run on win32 before?

Reply:
I don't think so @smlambert may confirm this? Though eventually we should run perf tests on all platforms.

Reply:
Certainly never run on win32 machines in the open.  

Also, some benchmarks are marked as not applicable for windows, so they will simply be skipped, example:
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/performance/idle_micro/playlist.xml#L27  (note: the ^os.win, not windows).

Reply:
We can fix as need be - a good goal is to have all platforms running the same test suite and just excluding tests that are irrelevant or need fixing.